datalake/src/main/java/com/sequenceiq/datalake/service/sdx/database/DatabaseService.java

```diff
@@ -118,6 +122,10 @@ public DatabaseServerStatusV4Response create(SdxCluster sdxCluster, DetailedEnvi
                 throw badRequestException;
             }
         }
+        if (serverStatusV4Response != null && serverStatusV4Response.getStatus().equals(Status.STOPPED)) {
+            throw new CloudbreakServiceException(String.format("Database already in %s state. Provisioning new database failed",
```

Comment:
When would we hit this scenario? Will throwing an exception here interrupt the resize operation, since it can't create the database?

Reply:
If this situation is hit the polling task is will never complete and will eventually timeout. we were hitting this scenario in the re-size scenario but with the changes are done in Detach flow, we are avoiding this scenario from occurring for resize use case.

Reply:
Will the polling task timeout, or will it just stop polling and fail with an error? I'm assuming throwing an exception will just end whatever operation is happening.

Reply:
Without this change the polling task will timeout after one hour.

Reply:
Throwing an exception basically would cause the creation of an external database to fail.